pages/app-layout/with-drawers.page.tsx

```diff
@@ -45,7 +38,13 @@ export default function WithDrawers() {
               content: <Security />,
               id: 'security',
               resizable: true,
-              size: widths.security,
+              initialSize: 500,
+              onResize: ({ size }: { size: number }) => {
```

Comment:
I think, this won't work, because the actual value is `event.detail.size`.

I think, this approach (without `detail`) is preferred for this case

Reply:
Yep! I think there might be another place with a similar issue :(. Hidden types eh?